### ISSUE TITLE
2.9 develop: stateswitch changes even if item doesn't have specific value

### ISSUE THREAD

**Issue by onkelandy** [Martin-Gleiss/smartvisu#159]
This behaviour might be intentional but I would love to see a way to suppress it:

When you have two stateswitch buttons for the same item, let's say one has 0, 50 and the other 0,100

As soon as you hit one of them, e.g. 100, the other one also changes.

Edit: changed icon to item

**Comment by smaiLee** [comment]
I'm sorry, I do not understand your report.

Do you mean "item" instead of "icon" or are you talking about dynamic icons?

The basic.stateswitch always represents the current value of an item. So yes, if the value of an item changes, the stateswitch is changed as well.
If the item has a value that is not part of the value list of realted stateswitch, the last defined state (icon, color) will be shown.

So how do you expect a stateswitch to show up, when the item's value does not match?
If it should show like 0-state then the following code works:
```
{{ basic.stateswitch(id, item, '', [50,0], ['icon50.svg', 'icon0.svg']) }}
{{ basic.stateswitch(id, item, '', [100,0], ['icon100.svg', 'icon0.svg']) }}
```
If you click on first switch, the value will change to 50. So the first switch shows icon50.svg, the second one icon0.svg.
If you click the second one, the value changes to 100, the first switch shows icon0.svg and the second one icon100.svg.

**Comment by onkelandy** [comment]
Perfect.. thanks for the information. So the order of the values is important and it looks like it's advisable to have the 0 value last. Because the other way round with
```
{{ basic.stateswitch(id, item, '', [0,50], ['icon0.svg', 'icon50.svg']) }}
{{ basic.stateswitch(id, item, '', [0,100], ['icon0.svg', 'icon100.svg']) }}
```
clicking on the "50" results in the second button showing "100" .. what was weird ;) Maybe you could add the sentence in the documentation? Thanks a lot!

"If the item has a value that is not part of the value list of realted stateswitch, the last defined state (icon, color) will be shown."

**Comment by smaiLee** [comment]
> Maybe you could add the sentence in the documentation?
Yes, I will do this, I had the same thought as I wrote my answer above :smile: 

I can tell some more to the reason why the last and not - as possibly expected - the first one is default:
Initially it was the first one, but then @psilo909 noticed that this behavior differs from basic.multistate in 2.8, which is superseded by basic.stateswitch.

**Comment by onkelandy** [comment]
ups, sorry.. I re-open so the "docu update" doesn't get lost.
